projects/thrift/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://thrift.apache.org/"
+language: c++
```

Comment:
@jonathanmetzman 
Thrift has 28 languages
I do not know if there should be some settings for these projects (ecc-diff-fuzzer is another one)

Reply:
How much code is there actually to fuzz in each language?
I assume the core code is written in one language and there are simply wrappers for other languages.
Maybe put the language of the one you are actually fuzzing.
If the core code is C++ why fuzz the go wrappers?

Reply:
> I assume the core code is written in one language and there are simply wrappers for other languages.

This is not the case for thrift, there are no wrappers.
Thrift compiles some schema into one of its 28 languages, and you get a RPC client/server only in this language (ie there is no C++ code run from golang), like 28 implementations of the same protocol(s)
So, there is about as much to fuzz in each language.

(There may also be the schema compiler, but it is itself rather a C++ parser around flex/bison )

Reply:
OK I misunderstood.
I've filed #5278 to discuss this.
What languages are we fuzzing for now? Just go?

Reply:
This draft PR has only golang indeed